### PR TITLE
feat: Added piste (direction) and time as factors for rating in data generation

### PR DESCRIPTION
**Pull request by TheMOHD3886**
# Another bomboclaat data-generation rework for da real generals 😎🇯🇲
Mock data now includes real piste ids and the direction of the piste being rated. Time and windeffect now included when calculating rating
## Changes
* savePistes function rework
    * Can now update existing entries in db
    * Added "direction" as field to each entry
* Time and windeffect are now included for calculating ratings
    * windeffect is based on wind-direction and the direction of the piste
* Added and fixed a lot of documentation

### CODE REVIEW

**Review by MortenMunk — APPROVED** [review]
LGTM 🥇

File: data_generation/index.js [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+	3: 0,
+	4: 0,
+	5: 0,
+};
+
+for (let pts in Object.keys(counts))  {
+	counts[Number(pts) + 1] = data.filter((rating) => rating.points === Number(pts) + 1).length;
+}
+
+console.log(counts);
```
Comment (Mafusn) [inline data_generation/index.js:19]: Jeg går amok

File: data_generation/utils/saveDataToFile.js [hunk 2]
```diff
@@ -2,12 +2,15 @@ import fs from 'fs';
...
  */
-function saveToFile(data, fileName) {
+function saveToFile(data, fileName, test = false) {
     // Write data to JSON file
     fs.writeFileSync(fileName, JSON.stringify(data, null, 2));
 
+    /* istanbul ignore next */
+    if (test) return;
+    /* istanbul ignore next */
     console.log('Data saved to ' + fileName);
```
Comment (Mafusn) [inline data_generation/utils/saveDataToFile.js:14]: Ik igen mand

File: population/index.js
Comment (Mafusn) [inline population/index.js:1]: Der er et par console.logs i den her fil

File: routes/ratingRoutes.js [hunk 3]
```diff
@@ -79,6 +79,7 @@ async (req, res) => {
 		await newRating.save();
 		res.status(200).send("Successfully rated piste");
 	} catch (error) {
+		console.log(error);
```
Comment (Mafusn) [inline routes/ratingRoutes.js:82]: Noooooo

**Review by Mafusn — COMMENTED** [review]
Only need to get rid of a couple console.log

**Review by Mafusn — APPROVED** [review]
Me so happy

**Pull request merged**